fdbserver/storageserver.actor.cpp

```diff
@@ -529,6 +543,11 @@ struct StorageServer {
 	ActorCollection actors;
 
 	StorageServerMetrics metrics;
+	// Unlike StorageServerMetrics, the counters below count the number of bytes writen between two "StorageMetrics"
+	// trace events, without information about key distributions.
+	int fetchKeysBytes = 0;
+	int updateMutationBytes = 0;
```

Comment:
Can these use the `Counter` type?

Reply:
I did't notice there was such thing. Done and consolidate it with `counters`.